package.json

```diff
@@ -87,6 +87,7 @@
     "sourceMap": false,
     "instrument": false
   },
+  "homepage": "https://mozilla.github.io/nunjucks/",
```

Comment:
This could go into a separate PR, that might get merged faster

Reply:
Probably, but maintainer got no time at that moment, he promise to sort out PRs later.

Reply:
If it's a one-liner, then it's much more likely to get merged without delay

Reply:
Moved to #1299